### PR TITLE
Cleanup unused code artifacts

### PR DESCRIPTION
**Pull request by chef-louis**
#24

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = {
         skyblue: "#00C5F4",
         primary: "#FDFEFF",
         bgnav: "rgba(0, 8, 15, 0.10)",
-        matchtittles: "rgba(0, 197, 244, 0.05)",
+        matchtitles: "rgba(0, 197, 244, 0.05)",
```
Comment (qisforq) [inline tailwind.config.js:31]: Nice catch!

File: src/pages/index.tsx [hunk 2]
```diff
@@ -31,23 +30,21 @@ interface NewsProps {
...
   // useEffect hook to handle preloader and body overflow
   useEffect(() => {
-   
     // Delay for preloader to disappear
     setTimeout(() => {
       setpreloader(false);
     }, 1500);
 
     // Adding CSS classes to body for overflow control
-    document.body.classList.add("overflow_anchor");
```
Comment (qisforq) [inline src/pages/index.tsx:50]: why was this line deleted?

**Review by qisforq — APPROVED** [review]
Everything here looks good to me. I just left one comment for clarity on why one line was deleted, otherwise it looks good to go

File: src/pages/index.tsx (hunk 2 above)
Comment (chef-louis) [reply src/pages/index.tsx:50]: the class "overflow_anchor" is not defined anywhere. So this line doesn't change anything with the application's rendered styling. @HectorAgudelo was this a legacy class that is no longer used?

Comment (qisforq) [reply src/pages/index.tsx:50]: Ah got it. Thanks for clarifying. that must have disappeared at some point.

Comment (qisforq) [reply src/pages/index.tsx:50]: I'll merge this PR now
